platform/macos/macos.xcodeproj/xcshareddata/xcschemes/CI.xcscheme

```diff
@@ -89,15 +117,16 @@
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES">
-      <MacroExpansion>
+      <BuildableProductRunnable
+         runnableDebuggingMode = "0">
          <BuildableReference
             BuildableIdentifier = "primary"
-            BlueprintIdentifier = "DA839E911CC2E3400062CAFB"
-            BuildableName = "Mapbox GL.app"
-            BlueprintName = "macosapp"
-            ReferencedContainer = "container:macos.xcodeproj">
+            BlueprintIdentifier = "8E8A234D3E364CDFA2918983"
+            BuildableName = "mbgl-test"
```

Comment:
Won’t mbgl-test [still run](https://github.com/mapbox/mapbox-gl-native/blob/04cf3ed05c4b95603ff11109a201d381242b16e1/Makefile#L128) without changing the runnable to mbgl-test? It’d be more convenient for development purposes for the runnable to be macosapp.